testdata/scripts/convert_complete_swagger.txt

```diff
@@ -174,7 +174,7 @@ service Service {
 //                 Version: "1.0",
 //         },
 //         Host:     "brank.as",
-//         BasePath: "/v1",
+//         BasePath: "/path",
 //         Schemes: []openapiv2.SwaggerScheme{
 //                 openapiv2.HTTP,
 //                 openapiv2.HTTPS,
```

Comment:
you can see it here - it can have more schemes, but we want to print `https` in the example if we can?